ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -18,8 +20,21 @@ import Ouroboros.Network.BlockFetch (FetchMode (..))
 import Ouroboros.Network.Diffusion.Policies (closeConnectionTimeout)
 import Ouroboros.Network.PeerSelection.Bootstrap (UseBootstrapPeers (..))
 import Ouroboros.Network.PeerSelection.Governor.Types hiding (targets)
+import Ouroboros.Network.PeerSelection.LedgerPeers (LedgerStateJudgement (..))
 import Ouroboros.Network.PeerSelection.PeerMetric
 
+-- | Tag indicating churning approach
+-- There are three syncing methods that networking layer supports, the legacy
+-- method with or without bootstrap peers, and the Genesis method that relies
+-- on chain skipping optimization courtesy of consensus, which also provides
+-- probabilistic ledger guarantees.
+-- Default - churn peers normally when syncing up in Genesis, or
+--  when we are caught up.
+-- ChurnLegacySync - mode used when we are behind and we are not using bootstrap
+--  peers.ActivityType
+-- ChurnBootstrapSync - same as above but with bootstrap peers
```

Comment:
I had a little dilemma what to name the constructors and had a few iterations, the reason why I picked Default is that this name applies to both Praos and Genesis where we are calculating the percentages of peers to drop. ChurnGenesis might be a little confusing when looking at functions such as decreaseActivePeers, because one might wonder why the first branch should be taken for Praos mode where the constructor might allude that that branch should be taken in Genesis mode.

Reply:
I see, then `ChurnDefault` will be more consistent with other constructors, and easier to understand in the use cases.